docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -3,15 +3,15 @@
 ## Develop
 
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites).
-- From a terminal, `npx create-react-app your-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`
+- From a terminal, `npx create-react-app your-app-name --template @itwin/web-viewer --scripts-version @bentley/react-scripts`
   - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
 - Open the `your-app-name` directory in VS Code.
 - Add a valid contextId (i.e. Project Id) and iModelId for your user to the IMJS_CONTEXT_ID and IMJS_IMODEL_ID variables in the .env file within the application's root directory.
   - You can obtain these ids from the "Show Ids" column of your "[My sample iModels](https://developer.bentley.com/my-imodels/)" page.
   - This will be used for initial development. The idea is that it would be replaced by a proper model selection process in a production application.
 - From a terminal at your application's root directory, `npm start`. This will serve the application with live reloading.
 - Add/Update/Remove files as needed for your use case. If running `npm start` while making changes, your application will recompile and reload.
-- The viewer can be modified via the Viewer component in the App.tsx file. Visit the [iTwin Viewer React](https://www.npmjs.com/package/@bentley/itwin-viewer-react) documentation for more information.
+- The viewer can be modified via the Viewer component in the App.tsx file. Visit the [iTwin Viewer React](https://www.npmjs.com/package/@itwin/web-viewer-react) documentation for more information.

```

Comment:
should we be more explicit and mention that this is the web viewer? if so, should probably update everywhere
```suggestion
- The viewer can be modified via the Viewer component in the App.tsx file. Visit the [iTwin Web Viewer React](https://www.npmjs.com/package/@itwin/web-viewer-react) documentation for more information.
```